tests/queries/0_stateless/02367_join_pushdown_column_not_found.sql

```diff
@@ -0,0 +1,9 @@
+SELECT deleted
```

Comment:
It'd be nice to have a test with more than one pushed down conditions, for right side and both as well (left and right)

Reply:
Added check for different algorithms https://github.com/ClickHouse/ClickHouse/blob/298fb2431da4a1ee85495d63557d405d9fe21339/tests/queries/0_stateless/02367_join_pushdown_column_not_found.sql.j2#L1

Reply:
Thanks. But it doesn't check right side of join